src/EditorFeatures/TestUtilities/LanguageServer/AbstractLanguageServerProtocolTests.cs

```diff
@@ -222,20 +223,21 @@ protected static LSP.CompletionParams CreateCompletionParams(LSP.Location caret,
         protected static LSP.VSCompletionItem CreateCompletionItem(
             string text, LSP.CompletionItemKind kind, string[] tags,
             LSP.CompletionParams requestParameters, bool preselect = false,
-            string[]? commitCharacters = null)
+            string[]? commitCharacters = null, string? sortText = null)
             => new LSP.VSCompletionItem()
             {
                 FilterText = text,
                 InsertText = text,
```

Comment:
if they support insertiontext, why are we not passing that along ourselves?  it is something we compute.

Reply:
renamed - we do pass the insertion text.